README.md

```diff
@@ -13,8 +13,10 @@ cd plotIt/
 
 # Initialize the git remotes
 source firstsetup.sh 
-# Load the proper environment (a la CMSSW_7_4_10)
-source setup_sl6_env.sh
```

Comment:
sure, I'd just prefer the 'default' install to be within CMSSW... I just remove it from the readme, I did not touch the script ?

Equally I could mention this script in a comment in the README itself, I don't mind